spec.html

```diff
@@ -6769,8 +6769,8 @@ <h1>GetSuperBase ( )</h1>
             1. Let _envRec_ be the function Environment Record for which the method was invoked.
             1. Let _home_ be _envRec_.[[HomeObject]].
             1. If _home_ has the value *undefined*, return *undefined*.
-            1. Assert: Type(_home_) is Object.
-            1. Return ? _home_.[[GetPrototypeOf]]().
+            1. Assert: _home_ is an ordinary object.
```

Comment:
how would this be compatible with `class extends Array { }`, for example?

Reply:
the home object is always an ordinary object (`proto` in ClassDefinitionEvaluation and `obj` in the ObjectLiteral RS) or an ecmascript function (`F` in ClassDefinitionEvaluation). The assertion needs to be updated to reflect the latter.

Reply:
This wouldn't hold true with the (removed prior to ES6) `Function.setHomeObject`method; however, it seems like a reasonable assertion to make (along with a non-normative note indicating this was not a design decision)

Reply:
@devsnek please note that [ECMAScript function object](https://tc39.es/ecma262/#sec-ecmascript-function-objects) is stated as an _ordinary object_, while [_ordinary object definition_](https://tc39.es/ecma262/#ordinary-object) holds for objects with ordinary `[[Call]]` & `[[Construct]]` methods.